src/Infrastructure/Services/ParticipantService.cs

```diff
@@ -181,25 +194,25 @@ public async Task DeleteUserAsync(string participantId)
             var entity = await _participantRepository.GetParticipantDetailsAsync(participantId);
             if (entity == null) return;
 
-            var baseUrl = _emailSettings.WebAppBaseUrl;
-            var htmlBody = EmailTemplate.GetHtmlTemplate().Replace("###TITLE_REPLACE1###",
-                    "Closure of Be Part of Research Account")
-                .Replace("###TEXT_REPLACE1###",
-                    "This email is to confirm the closure of your Be Part of Research account. If you would still like to hear from us, you can<a href=\"https://nihr.us14.list-manage.com/subscribe?u=299dc02111e8a68172029095f&id=3b030a1027\"> sign up to our newsletter</a> to receive all our research news, and hear about studies and other opportunities to help shape health and care research from across the UK.")
-                .Replace("###TEXT_REPLACE2###",
-                    "If you would like to register again in future, please visit the <a href=\"https://volunteer.bepartofresearch.nihr.ac.uk/Participants/introduction\">registration page</a>.")
-                 .Replace("###TEXT_REPLACE3###",
-                    "Thank you for your support.")
-                .Replace("###TEXT_REPLACE4###",
-                    "")
-                .Replace("###LINK_REPLACE###", "")
-                .Replace("###LINK_DISPLAY_VALUE_REPLACE###", "block")
-                .Replace("###TEXT_REPLACE5###",
-                    "")
-                .Replace("###TEXT_REPLACE6###",
-                    "");
-
-            await _emailService.SendEmailAsync(entity.Email, "Be Part of Research", htmlBody);
+            // var baseUrl = _emailSettings.WebAppBaseUrl;
```

Comment:
Delete these commented out lines. They are already in source control if you need to recover them.